src/bindings/js/node/lib/addon.ts

```diff
@@ -12,10 +12,24 @@ type SupportedTypedArray =
   | Float64Array;
 
 interface Core {
-  compileModel(model: Model, device: string, config?: {[option: string]: string}): CompiledModel;
-  compileModelSync(model: Model, device: string, config?: {[option: string]: string}): CompiledModel;
-  readModel(modelPath: string, binPath?: string): Promise<Model>;
-  readModelSync(modelPath: string, binPath?: string): Model;
+  compileModel(
+    model: Model,
+    device: string,
+    config?: { [option: string]: string }
+  ): Promise<CompiledModel>;
+  compileModelSync(
+    model: Model,
+    device: string,
+    config?: { [option: string]: string }
+  ): CompiledModel;
+  readModel(
+    modelPath: string | Uint8Array,
+    binPath?: string | Uint8Array,
+  ): Promise<Model>;
+  readModelSync(
+    modelPath: string | Uint8Array,
```

Comment:
modelPath or modelBuffer?
```suggestion
   pathOrBuffer: string | Uint8Array,
```